packages/dcos-diagnostics/extra/dcos-diagnostics-agent.service

```diff
@@ -4,12 +4,16 @@ Description=DC/OS Diagnostics Agent: exposes component health
 EnvironmentFile=/opt/mesosphere/environment
```

Comment:
Why are any changes to this file necessary now?

Reply:
Please read commit message: https://github.com/dcos/dcos/pull/6032/commits/727cd76eaae050c87ee960d1adce7377da4f4b78